commons/src/types/Availability.ts

```diff
@@ -29,7 +29,6 @@ export interface Manifest extends NylasManifest {
   event_buffer: number;
   free_color: string;
   mandate_top_of_hour: boolean;
-  max_bookable_slots: number;
```

Comment:
Is this just the availability manifest max_bookable_slots? Or does this also need to be removed from ScheduleEditor/Booking?

Reply:
Good point, I'll remove it everywhere